### PR TITLE
Agdasima: Version 2.002 added

### PR DESCRIPTION
**Pull request by yanone**
6f00c87: [gftools-packager] Agdasima: Version 2.002 added

* Agdasima Version 2.002 taken from the upstream repo https://github.com/docrepair-fonts/agdasima-fonts at commit https://github.com/docrepair-fonts/agdasima-fonts/commit/6cb88a3dfae8073b51b83c27049bb4552bbecdf4.

### CODE REVIEW

**Comment by yanone** [comment]
Updated

## Agdasima: Version 2.002 added

---
 9eb64a1: [gftools-packager] Agdasima: Version 2.002 added

* Agdasima Version 2.002 taken from the upstream repo https://github.com/docrepair-fonts/agdasima-fonts at commit https://github.com/docrepair-fonts/agdasima-fonts/commit/86b8456957d1c250a613e8c763634e562f898021.

**Comment by yanone** [comment]
Updated

## Agdasima: Version 2.002 added

---
 6a9eca9: [gftools-packager] Agdasima: Version 2.002 added

* Agdasima Version 2.002 taken from the upstream repo https://github.com/docrepair-fonts/agdasima-fonts at commit https://github.com/docrepair-fonts/agdasima-fonts/commit/86b8456957d1c250a613e8c763634e562f898021.

**Comment by yanone** [comment]
Updated

## Agdasima: Version 2.002 added

---
 6c0af25: [gftools-packager] Agdasima: Version 2.002 added

* Agdasima Version 2.002 taken from the upstream repo https://github.com/docrepair-fonts/agdasima-fonts at commit https://github.com/docrepair-fonts/agdasima-fonts/commit/affd96dca1cc5e00c8590669c74e7f21e8c35f0d.

**Comment by m4rc1e** [comment]
The asterisk feels too wide. I've seen worse though so not a blocker as far as I'm concerned.

Screenshot 2023-04-03 at 23 45 34

**Comment by m4rc1e** [comment]
@twardoch we also have the following fail:

**Ensure soft_dotted characters lose their dot when combined with marks that replace the dot.**

**Comment by yanone** [comment]
I would plea to ignore this, as the check first of all doesn't check whether the language support of that font even makes any sense to have those glyphs supported, and secondly the DR fonts come as exact replicas of their reference fonts in terms of glyph support, and OT features: they don't have any, and the soft-dotted-i issue requires ccmp.

**Review by twardoch — APPROVED** [review]

File: ofl/agdasima/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<p>
+    Agdasima is a DocRepair font, intended to improve compliance with the ISO/IEC
+    29500 standard by providing fallback that minimizes text reflow in Office Open
+    XML documents. Agdasima is based on Big Shoulders, a condensed American Gothic
+    sans-serif font family.
+</p>
```
Comment (RosaWagner) [inline ofl/agdasima/DESCRIPTION.en_us.html:6]: @yanone no github repo? is this something we should ignore for all docrepair fonts?

Comment (yanone) [reply ofl/agdasima/DESCRIPTION.en_us.html:6]: Adam says they're not open for contributions. I agree.

**Comment by RosaWagner** [comment]
@twardoch the description doesn't say which font it is "replacing"

**Comment by yanone** [comment]
This is the PR that Adam accidentally merged. I believe that addition came in a later PR, which Marc also already merged. So all subsequent changes will have to be new PRs as well.

**Comment by RosaWagner** [comment]
The second PR #6180 was adding the URL of the upstream repo. Yes indeed, a new PR is needed to correct the description.

**Comment by yanone** [comment]
Is anything missing here other than the font replacement in the description that was found while I was away?

**Comment by yanone** [comment]
Update: Adam did change the asterisk, so I'll re-PR the whole thing.

**Comment by yanone** [comment]
New PR: #6249

**Pull request merged**